i18n/README.es-es.md

```diff
@@ -105,15 +135,17 @@ de filas, corriendo en un servidor `c5.metal` usando 16 de los 96 hilos disponib
 
 ### 📚 Lee la documentación
 
-- [Documentación de QuestDB:](https://questdb.io/docs/introduction/) entiende cómo instalar, configurar y correr QuestDB.
+- [Documentación de QuestDB:](https://questdb.io/docs/introduction/) entiende cómo instalar, configurar y correr
+  QuestDB.
 - [Tutoriales:](https://questdb.io/tutorial/) aprende paso a paso todo lo que puedes hacer con QuestDB.
-- [Product roadmap:](https://github.com/questdb/questdb/projects) echa un vistazo a nuestro plan para las versiones que vienen.
+- [Product roadmap:](https://github.com/questdb/questdb/projects) echa un vistazo a nuestro plan para las versiones que
```

Comment:
do all links need to end in slash?, if so projects seems missing it. I am just commenting because my understanding is that this is relevant for SEO

Reply:
I don't think it is needed. Maybe @argshook knows better?

Reply:
All https://questdb.io links should include a trailing slash